lib/strategy.js

```diff
@@ -387,8 +386,12 @@ Strategy.prototype.configure = function(identifier, done) {
  * @return {Object}
  * @api protected
  */
-Strategy.prototype.authorizationParams = function(options) {
-  return {};
+AcpStrategy.prototype.authorizationParams = function(options) {
```

Comment:
had to stick with ES5 for the package, so spread operator isn't supported. also, we could add other params to the `params` object later on, so that's why the few extra lines.